tests/network/placeholder.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+
+package network
```

Comment:
What is the advantage to place it under a new package?
I understand why helpers merit their own packages (supporting dependencies between tests and helpers).

Reply:
It limits the scope for any helper functions if we will introduce them. It also clearly identifies a subset of tests that are testing networking. We can have our own OWNER file.

In Kubernernetes, they follow similar structure where each SIG has its own subdirectory for tests.

Reply:
We can add a suffix. But having directories with tests and directories with utils mixed in the same directory smells. The structure similar to what we use in other projects would be:

```
tests/
| - e2e/
|   | - e2e_suite.go
|   | - network/
|       | - migration.go
| - name_of_your_utils
    | - ...
```

Maybe we should adapt that instead. However, it would require us to move all the code. I'd wait on that.